packages/dara-components/dara/components/smart/code_editor/code_editor.py

```diff
@@ -30,5 +32,16 @@ class CodeEditor(StyledComponentInstance):
 
     script: NonDataVariable
 
+    language: Optional[
+        Union[
+            Literal['json'],
+            Literal['python'],
+            Literal['markdown'],
+            Literal['sql'],
+            None,
+        ]
+    ] = None
```

Comment:
You don't need both the `None` within the `Union` and `Optional`. `Optional` is `Union[T, None]` under the hood